UnitySDK/Assets/ML-Agents/Scripts/DemonstrationRecorder.cs

```diff
@@ -37,14 +40,16 @@ private void Update()
         /// </summary>
         private void InitializeDemoStore()
         {
-            recordingAgent = GetComponent<Agent>();
-            demoStore = new DemonstrationStore();
+            // ReSharper disable once Unity.PerformanceCriticalCodeInvocation
```

Comment:
Any idea what's up with these comments?

Reply:
ah yeah, I can remove those.  There are performance indicators that show up in rider as warning-like annotations.

Reply:
these comments prevent them from popping up as warnings, but is also just noise in the code.